### PR TITLE
Added QEMU ubuntu 22.04 build

### PR DESCRIPTION
**Pull request by AverageMarcus**
What this PR does / why we need it: Adds Ubuntu 22.04 builds for QEMU platform

Which issue(s) this PR fixes (optional, in fixes #<issue number>(, fixes #<issue_number>, ...) format, will close the issue(s) when PR gets merged): Fixes #

**Additional context**
Currently untested / WiP

### CODE REVIEW

File: images/capi/packer/ova/linux/ubuntu/http/22.04/preseed-efi.cfg [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2019 The Kubernetes Authors.
```
Comment (tobiasgiese) [inline images/capi/packer/ova/linux/ubuntu/http/22.04/preseed-efi.cfg:1]:
```suggestion
# Copyright 2022 The Kubernetes Authors.
```

We should update the year for new files

File: images/capi/packer/ova/linux/ubuntu/http/22.04/preseed.cfg [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2019 The Kubernetes Authors.
```
Comment (tobiasgiese) [inline images/capi/packer/ova/linux/ubuntu/http/22.04/preseed.cfg:1]:
```suggestion
# Copyright 2022 The Kubernetes Authors.
```

And this file as well 🙂

**Comment by AverageMarcus** [comment]
Closing in favour of https://github.com/kubernetes-sigs/image-builder/pull/997

**Pull request closed without merging**